vagrant/ansible/roles/ctdb.setup/tasks/main.yml

```diff
@@ -0,0 +1,79 @@
+---
+- name: Create the ctdb replicated volume
+  gluster_volume:
+    state: present
+    name: "{{ gluster_features_ctdb_volume }}"
+    bricks: '{{ ctdb_brick_location }}/ctdb'
+    cluster: "{{ gluster_cluster_hosts }}"
+    replicas: "{{ gluster_cluster_replica_count }}"
+  run_once: yes
+
+- name: Add ctdb package
+  yum:
+    name: ctdb
+    state: present
+
+- name: Add ctdb rules to firewalld
+  firewalld:
+    service: ctdb
+    permanent: yes
+    state: present
+    state: enabled
+
+- name: Add recovery lock to ctdb.conf
+  lineinfile:
+    dest: /etc/ctdb/ctdb.conf
+    line: "\trecovery lock = /gluster/lock/recovery.lock"
+    regexp: "recovery lock"
+    insertafter: '\[cluster\]'
+
+- name: Add netbios name - The two ctdb nodes should have the same netbios name
+  lineinfile:
```

Comment:
This should not be in the CTDB setup activities to modify an existing smb.conf file. We should have it in the smb.conf from the start, when setting up *Samba*, not *ctdb*. E.g use a template (jinja / j2).

Reply:
Suggested changes made and will shortly be uploaded. 
The gluster fuse mount is used to set root permission to 0777 to allow us to easy write test scripts.